README.md

```diff
@@ -53,28 +77,24 @@ After following the steps, run the command below and follow the instructions to
 ts-node src/google/create-token.ts
 ```
 
-## Deployment
+## Development
 
-Before deployment, create an `.env` file using the fields mentioned below:
+To add new features to the bot. Fork the repository and add your feature to `src/meetbot/features`. After finishing your changes, [run the bot locally](#running-the-bot-locally) to check if your feature loads and test your changes. 
 
-```
-GOOGLE_PASSWORD=              # Password for your bot/user that is used to join Google Meet. 
-GOOGLE_TOTP_SECRET=           # [Optional] If the bot account has 2FA auth, then add 2FA secret here.
-GOOGLE_LOGIN=                 # Email-ID/username of your bot/user
-TOKEN_PATH=token.json         # [Default] Path to token file 
-HTTP_PORT=80                  # Port to run express server on. For prod, use port 80.
-RUNNING_ALPINE=0              # [Default] If base image is alpine then set the env var, otherwise don't. 
-```
+To develop the UI, run `npm run dev-ui`. This will spin up the development server. You should then be able to access the dashboard at `http://localhost:3000/`. Note that the data you will see on the tables are actually being mocked by MirageJS, and not actual request is going to the API. This makes it easier to develop both components in an entirely decoupled way.
 
-Run the command below and follow the instructions to generate a `token.json` file. This authentication process is one time only.
+## Deployment
 
-```
-ts-node src/google/create-token.ts
-```
+To activate all the features of meetbot, [authenticate the meetbot](#authentication) and create the `token.json` file before deployment. 
 
-To deploy, use the dockerfiles present in the repository. Tweak the base image and other details as per your deployment platform (AWS, Heroku etc.) and deploy. 
-To deploy on balenaCloud, run the command below:
+To deploy on balenaCloud, use the balena CLI and balena push command as stated below. For more information [check out the docs](https://www.balena.io/docs/learn/deploy/deployment/).
 
 ```
 balena push <Name of fleet>
 ```
+
+To deploy on a server, create and fill the `.env` file in the root of the project directory using the `.env.example` file and deploy using the Dockerfile present in the root directory of this repository. 
```

Comment:
Is this still correct?

Reply:
Indeed, that's the big issue. I need help over deployment instructions. Maybe we can have a call to improve this. 

Server would still need a .env file since they don't have a balena.yml to help them out. That's the big issue I am fighting. My .env file way was universal but the idea got taken apart in the clinic. I appreciate the feedback and folks were correct too. I don't like creating .env files since they are a big blocker. Atleast balena.yml gives you a nice interface to enter the value on deploy with balena.

Reply:
I'm working (very low prio) on an extension to the [local UI block](https://github.com/balenablocks/local-ui) to give the user the option to edit env-vars on the device. For that I parse the compose file to get the list of defined env-vars. That's something balena cloud could do, too to give a nicer experience.

As a super simple first step, we could have the UI show a message "you need to set the following env vars..." until the server has validated that all necessary env-vars have been set